indico/modules/events/layout/controllers/images.py

```diff
@@ -18,11 +18,14 @@
 
 import shutil
 from io import BytesIO
+from mimetypes import guess_type
 
 from flask import flash, render_template, request, session
 from PIL import Image
+from pyPdf import PdfFileReader
```

Comment:
this adds a new dependency, with the last release being 7 years old. I don't think we want this in indico.